genie-web/src/test/java/com/netflix/genie/web/tasks/leader/ClusterCheckerTaskUnitTests.java

```diff
@@ -115,7 +119,28 @@ public void canRun() throws IOException, GenieException {
             .thenThrow(new RestClientException("blah"))
             .thenReturn("")
             .thenReturn("")
-            .thenThrow(new RestClientException("blah"))
+            .thenThrow(new HttpServerErrorException(HttpStatus.INTERNAL_SERVER_ERROR, "",
+                ("{\"status\":\"OUT_OF_SERVICE\", \"genie\": { \"status\": \"OUT_OF_SERVICE\"}, "
```

Comment:
Add tests for other status cases other than just OUT_OF_SERVICE... like DOWN or UNKNOWN